lib/kafka_ex/config.ex

```diff
@@ -80,6 +80,7 @@ defmodule KafkaEx.Config do
 
   defp server("0.8.0"), do: KafkaEx.Server0P8P0
   defp server("0.8.2"), do: KafkaEx.Server0P8P2
+  defp server("0.10.1"), do: KafkaEx.Server0P10P1
```

Comment:
you want to move 0.9.0 up and make this the default

Reply:
ok, will do